docs/source/get_started/install.md

```diff
@@ -43,7 +43,9 @@ We suggest you create a new Python virtual environment for *each* new Kedro proj
 [Read more about Python virtual environments](https://realpython.com/python-virtual-environments-a-primer/) or [watch an explainer video about them](https://youtu.be/YKfAwIItO7M).
 ```
 
-Depending on your preferred Python installation, you can alternatively create virtual environments to work with Kedro using `venv` or `pipenv` instead of `conda`, as described in the [FAQ](../faq/faq.md)
+```{note}
+Depending on your preferred Python installation, you can alternatively create virtual environments to work with Kedro using `venv` or `pipenv` instead of `conda`.
```

Comment:
xref https://github.com/kedro-org/kedro/issues/2360#issuecomment-1447971885

Just for the record, do you intend to add such instructions in this PR?

Reply:
I wasn't planning to because #2360 is coming (we should prioritise for next sprint) and we'll have both PRs in the docs before next release, so the absence should be OK.